packages/web3/src/index.ts

```diff
@@ -321,6 +321,41 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 /**
  * This comment _supports3_ [Markdown](https://marked.js.org/)
  */
+
 import Web3 from './web3';
 
-export = Web3;
+export default Web3;
+
+/**
```

Comment:
This will cause breaking change :
`const Web3 = require('web3').default;` instead of `const Web3 = require('web3');`
and I prefer if we can avoid it and as solution shared  by Marin we can use [conditional export / separate entry point](https://nodejs.org/api/packages.html#conditional-exports).

Reply:
For the sake of our users, we should do that change as importing like this is such a bad habbit and antipattern that everyone should avoid in modern js..